api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -138,8 +129,7 @@ message ClusterStats {
   //
   // .. code-block:: none
   //
-  //   sum_locality(total_successful_requests) + sum_locality(total_requests_in_progress) +
-  //     sum_locality(total_error_requests) + total_dropped_requests`
+  //   sum_locality(total_issued_requests) + total_dropped_requests`
```

Comment:
Why was this kept but not at endpoint level?

Reply:
I let it be just for total_dropped_requests. Felt that total requests is the sum of issued and dropped was non trivial. But now that i think about it again, i can remove this as well. I defer to your judgment.

I removed it at locality and endpoint level because the previous comments were partially incorrect and total_issued_requests field is self explanatory.

Reply:
triggered a coverage run with logging...

Reply:
Let me know if I need to change something here.

Reply:
I would drop this for consistency. Cluster and endpoint level should match IMHO.

Reply:
Done. PTALA.